src/script/components/MessagesList/Message/MemberMessage.tsx

```diff
@@ -109,10 +102,8 @@ export const MemberMessage: React.FC<MemberMessageProps> = ({
             {isMemberRemoval && <span className="icon-minus" />}
             {isMemberJoin && <span className="icon-plus" />}
           </div>
-          {/* event is being triggered only when clicked on <a> tag with specified class (keyboard accessible by default) */}
-          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
           <div className="message-header-label">
-            <MessageContent onClickAction={() => onClickParticipants(highlightedUsers)} message={message} />
+            <MessageContent onClickParticipants={participants => onClickParticipants(participants)} message={message} />
```

Comment:
You can simplify it :)

```suggestion
            <MessageContent onClickParticipants={onClickParticipants} message={message} />
```